docs/view-managers.md

```diff
@@ -237,6 +237,41 @@ This example assumes that the `ViewManagerSample.ReactPackageProvider` we create
 
 The `Microsoft.ReactNative.Managed.ReactPackageProvider` is a convenience that makes sure that all native modules and view managers defined within the app project automatically get registered. So if you're creating your view managers directly within the app project, you won't actually want to define a separate `ReactPackageProvider`.
 
+### More extensibility points
+
+- In some scenarios, a view manager might need to have more context at control creation time in order to decide what kind of control to instantiate.
+This is possible by having the view manager implement the `IViewManagerCreateWithProperties` interface. The `CreateWithProperties` method can then access the properties set in JSX by inspecting the `propertyMapReader`.

```

Comment:
nit, reword - 'made possible by' | 'achievable by' | 'possible via'

Reply:
```suggestion
This can be achieved by having the view manager implement the `IViewManagerCreateWithProperties` interface. The `CreateWithProperties` method can then access the properties set in JSX by inspecting the `propertyMapReader`.
```